core/trino-main/src/main/java/io/trino/cost/FilterStatsCalculator.java

```diff
@@ -119,7 +120,7 @@ private Expression simplifyExpression(Session session, Expression predicate, Typ
         // TODO reuse io.trino.sql.planner.iterative.rule.SimplifyExpressions.rewrite
 
         Map<NodeRef<Expression>, Type> expressionTypes = getExpressionTypes(plannerContext, session, predicate, types);
-        ExpressionInterpreter interpreter = new ExpressionInterpreter(predicate, plannerContext, session, expressionTypes);
+        ExpressionInterpreter interpreter = new ExpressionInterpreter(predicate, plannerContext, session, expressionTypes, Collections.emptyMap());
```

Comment:
nit: we use `ImmutableMap.of()` instead of `Collections.emptyMap()`

Reply:
replaced all `Collections.emptyMap()` to `ImmutableMap.of()` in the second commit.